### PR TITLE
commit on added-codes

### PR DESCRIPTION
**Pull request by HatimDev-HE**
In this feature, I made a Tribute page for Ms. Jane Dolphin

### CODE REVIEW

**Comment by HatimDev-HE** [comment]
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 🚀
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.**_**_

Code Review by [@HatimDev-HE ](https://github.com/HatimDev-HE)

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  position: relative;
+  bottom: 500px;
+  text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: bold;
+}
+
+main {
+  background: linear-gradient(90deg, rgba(162, 170, 172, 1) 0%, rgba(90, 91, 116, 1) 100%, rgba(2, 0, 36, 1) 100%);
+}
```
Comment (Ginohmk) [inline styles.css:22]: - Please make use of inline comments to point out areas that needed to be changed in the code.

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi @HatimDev-HE , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- Good summary for PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Suggestions **Note**

- [ ]  Please make use of inline comments and have at least to request changes to be made.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# Project Approved 🚀

Hi @HatimDev-HE   🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights 
- You included an explanation, a "why", for each requested change 🎉 
-  You phrased the comments politely, as suggestions instead of orders 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Kindly include emojis and gifs to make your code review more human-friendly 🙂 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**